style-guide/index.rst

```diff
@@ -33,6 +33,15 @@ customers.
    Provides instructions for when to use and how to create screenshots and
    diagrams to enhance your content.
 
+:ref:`how-to-article-guidelines`
+   In addition to the other guidelines in this style guide, some of which are
```

Comment:
I think there's something amiss in this first sentence here.

Reply:
Fixed